### PR TITLE
Package.json cleanup

### PR DESCRIPTION
**Pull request by nils-keller-dev**

### CODE REVIEW

**Comment by nils-keller-dev** [comment]
[package.json cleanup](https://trello.com/c/cl5UzC1J/13-packagejson-cleanup)

File: package.json [hunk 1]
```diff
@@ -1,42 +1,42 @@
...
   "scripts": {
     "dev": "vite",
     "build": "vite build",
     "serve": "vite preview",
     "lint": "eslint . --ext .js,.jsx,.ts,.tsx",
-    "lint:fix": "eslint . --ext .js,.jsx,.ts,.tsx --fix",
-    "format": "prettier --check 'src/**/*.{js,jsx,ts,tsx,json,css,scss,md}'",
-    "format:fix": "prettier --write 'src/**/*.{js,jsx,ts,tsx,json,css,scss,md}'"
-
+    "lint:fix": "eslint . --ext .js,.jsx,.ts,.tsx --fix"
```
Comment (sebakocz) [inline package.json]: ok but how about `"lint:fix": "npm run lint -- --fix"` instead

File: vite.config.ts [hunk 2]
```diff
@@ -7,12 +7,12 @@ export default defineConfig({
       plugins: [
         //  Toggle the booleans here to enable / disable Phaser 3 features:
         replace({
-          'typeof CANVAS_RENDERER': "'true'",
-          'typeof WEBGL_RENDERER': "'true'",
-          'typeof EXPERIMENTAL': "'true'",
-          'typeof PLUGIN_CAMERA3D': "'false'",
-          'typeof PLUGIN_FBINSTANT': "'false'",
-          'typeof FEATURE_SOUND': "'true'",
```
Comment (sebakocz) [inline vite.config.ts:15]: ok no clue how to check impact of those changes, lets keep this in mind

**Review by sebakocz — CHANGES_REQUESTED** [review]
lookin good

expected things to break but seems to be still running, thanks for refreshing packages

File: package.json (hunk 1 above)
Comment (nils-keller-dev) [reply package.json]: i like it, pushed

**Pull request merged**